### PR TITLE
add until option in rebuild command

### PR DESCRIPTION
**Pull request by DavidBadura**

### CODE REVIEW

File: src/Pipeline/Pipeline.php [hunk 1]
```diff
@@ -36,13 +36,13 @@ public function run(?callable $observer = null): void
...
         foreach ($this->source->load() as $bucket) {
+            $observer($bucket);
+
             $result = ($this->middlewares)($bucket);
 
             foreach ($result as $resultBucket) {
                 $this->target->save($resultBucket);
             }
-
-            $observer($bucket);
```
Comment (DanielBadura) [inline src/Pipeline/Pipeline.php:45]: Do you know why this was executed after the middleware seems this was wrong from the beginning doesnt it ?

Comment (DavidBadura) [reply src/Pipeline/Pipeline.php:45]: I guess I didn't think about it. I only noticed it while editing.

File: src/Console/Command/ProjectionRebuildCommand.php [hunk 2]
```diff
@@ -34,7 +40,8 @@ protected function configure(): void
         $this
             ->setName('event-sourcing:projection:rebuild')
             ->setDescription('rebuild projection')
-            ->addOption('recreate', 'r', InputOption::VALUE_NONE, 'drop and create projections');
+            ->addOption('recreate', 'r', InputOption::VALUE_NONE, 'drop and create projections')
+            ->addOption('until', 'u', InputOption::VALUE_OPTIONAL, 'create the projection up to a point in time [2017-02-02 12:00]');
```
Comment (DanielBadura) [inline src/Console/Command/ProjectionRebuildCommand.php:44]: Value should be required if option is given

File: src/Pipeline/Pipeline.php (hunk 1 above)
Comment (DanielBadura) [reply src/Pipeline/Pipeline.php:45]: okey 👍

**Review by DanielBadura — APPROVED** [review]

**Pull request merged**